sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -112,6 +112,10 @@ def check_database(self):
 
         return db_exists, context
 
+    def check_database_conns(self, db_name):
+        self.open_db_connections(None, db_name=db_name, is_default=False)
+        self.close_db_connections(None, db_name)
```

Comment:
Is it ok to close the default connection if this is being called at the end of the check?

Reply:
We should try to maintain long running connections

Reply:
Thanks, updated to not close for default connections!

Reply:
isnt still calling `self.close_db_connections`?

Reply:
I stopped checking for the default database in the loop here: https://github.com/DataDog/integrations-core/pull/9900/files#diff-d489ea433e417068fc5a8b9b6dbc692b21ab9e559f885c987ece4c26bfe31322R456